pkg/translator/loki/logs_to_loki.go

```diff
@@ -141,6 +126,50 @@ func LogsToLokiRequests(ld plog.Logs) map[string]PushRequest {
 	return requests
 }
 
+type PushEntry struct {
+	Entry  *push.Entry
+	Labels model.LabelSet
+	Tenant string
+}
+
+func LogToLokiEntry(lr plog.LogRecord, rl pcommon.Resource, scope pcommon.InstrumentationScope) (*PushEntry, error) {
```

Comment:
Perhaps this deserves a new test? Or adapt the existing test to call this explicitly?

Reply:
yeah, it does :) added tests, thank you!